src/array_api_extra/testing.py

```diff
@@ -36,13 +39,22 @@ def override(func: object) -> object:
 _ufuncs_tags: dict[object, dict[str, Any]] = {}  # type: ignore[explicit-any]
 
 
+class Deprecated(enum.Enum):
+    """Unique type for deprecated parameters."""
+
+    DEPRECATED = 1
+
+
+DEPRECATED = Deprecated.DEPRECATED
```

Comment:
I couldn't find anything to help here in the standard library, and I find that odd. Did I miss anything? I was expecting something like `@functools.deprecated_args("static_argnames", "static_argnums")`.

Reply:
Good pointer; however it requires Python >=3.13 so I can't use it :disappointed:

Reply:
You could get it from [typing-extensions](https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.deprecated). It looks like `typing-extensions` is already a dependency too.

Reply:
It isn't. array-api-extra exclusively depends on array-api-compat exclusively, which in turn has no dependencies (not even numpy).

Reply:
I see, it's only in the lint dependencies, I skimmed too quickly.

https://github.com/data-apis/array-api-extra/blob/84b1f0b0c0eb11c3b1acd2bb9aa3f70bac429a37/pyproject.toml#L56-L57